### PR TITLE
Add unittests for the classifier.

### PR DESCRIPTION
**Pull request by samschaevitz**
- Checks for successful initialization
- Tests exist for known exceptional input file scenarios using pyfakefs

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+   "tabWidth": 2,
+   "singleQuote": false
```
Comment (bdittes) [inline .prettierrc.json]: Uwazi does single quotes, so I'd prefer to do the same here

File: tests/conftest.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 from app import create_app
 
+pytest_plugins = "pyfakefs"
```
Comment (bdittes) [inline tests/conftest.py]: prefer ("pyfakefs",)

File: tests/test_classifier.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        result = c.classify("Where is my medical book?", "test_model")
+
+        assert c.vocab is not None
+        assert c.embedder is not None
+        assert c.predictor is not None
+
+        assert c.instance == "test_instance"
+        print(result)
+        assert result
+        for topic in result:
```
Comment (bdittes) [inline tests/test_classifier.py]: prefer 'for topic, _ in result.items():'

File: tests/test_routes.py [hunk 4]
```diff
@@ -1,8 +1,16 @@
 import json
+import os
 import pytest
 
 
-def test_classify(app):
+def test_classify(app, fs):
+    base_classifier_path = "./testdata"
```
Comment (bdittes) [inline tests/test_routes.py]: this is used in several places (e.g. flask_test_cfg), why not use from there?

Comment (samschaevitz) [reply tests/test_routes.py]: true, done

File: .prettierrc.json (hunk 1 above)
Comment (samschaevitz) [reply .prettierrc.json]: done, but not getting it to work for json files.

**Pull request merged**